opacus/layers/dp_rnn.py

```diff
@@ -630,6 +630,24 @@ def initialize_cells(self):
         self.set_rename_map(rename_map)
         return cells
 
+    def parameters(self, recurse=True):
+        """
+        Redefining parameters() as the RenameParamsMixin confuses it
```

Comment:
sorry, confuses what?

Reply:
For some reason, RenameParamsMixin gives parameters() for both the DPRNN and the RNNLinear, which are the same. This is a way around it but there might be something cleaner?

Reply:
I don't know if there is a cleaner way; this implementation is clean enough TBH. My question was mostly around what "it" refers to in the docstring; elaborating it would be useful :)

Reply:
I found a way around that (basically not triggering grad samples on DPRNN/DPLSTM as it is done on their LSTMLinear), not needed anymore!